master/internal/resourcemanagers/priority.go

```diff
@@ -162,12 +162,7 @@ func trySchedulingTaskViaPreemption(
 	log.Debugf("trying to schedule task %s by preempting other tasks", allocationRequest.Name)
 
 	for priority := model.MaxUserSchedulingPriority; priority > allocationPriority; priority-- {
-		if _, ok := priorityToScheduledTaskMap[priority]; !ok {
-			continue
-		}
-
-		preemptionCandidates := priorityToScheduledTaskMap[priority]
-		for _, preemptionCandidate := range preemptionCandidates {
+		for _, preemptionCandidate := range priorityToScheduledTaskMap[priority] {
```

Comment:
Not needed for this change, but I couldn't help
 throwing in another bit of cleanup I noticed.

Reply:
Does this actually handle errors?

Reply:
Yep, if the key isn't present in the map, then the index expression will evaluate to `nil`, so the loop will just be a no-op.